### ISSUE TITLE
[Proposal] Setup a new SIG for Advocacy and Outreach

### ISSUE THREAD

**Issue by LinuxSuRen** [kubesphere/community#291]
# Background
Community growth is very important for any community. Advocacy and outreach can dramatically help with that. And the key point is that it should not be done by one person or privately.

As far as I can see, there's a couple of forks are interested in this area.

# Why
As an open-source community, we should discuss everything as possible as we can. Discuss advocacy and outreach in an open way is reasonable. Setup this kind of SIG can gather much more great ideas and innovation.

# Benefit
* Less private conversations
* A new channel for those who might want to join us
* As a possible bridge between the software engineers, community managers, and marketing
* Less unnecessary disturb to those who are not interested in this

# Who
Basically, anyone who is interested in the discussion around advocacy and outreach. No matter you're a software engineer, technical writer, or marketing, community growth needs you.

## Potential members
I guess these people might be the potential initial members:
* @shaowenchen 
* @FeynmanZhou 
* @JiannaLiu
* @LinuxSuRen 

# Concern
* There are not enough people to join this SIG.
  * 3 people should be enough to startup it

# Reference
https://www.jenkins.io/sigs/advocacy-and-outreach/

**Comment by FeynmanZhou** [comment]
Support! There should be a public SIG for Advocacy and Outreach in the KubeSphere community.

**Comment by FeynmanZhou** [comment]
Let's schedule a meeting to discuss how to start and operate this SIG.

Feynman is inviting you to a scheduled Zoom meeting.

Topic: Setup a new SIG for Advocacy and Outreach
Time: Feb 3, 2021 10:00 AM Beijing, Shanghai

Join Zoom Meeting
https://us02web.zoom.us/j/83700282246?pwd=NThOMTNqMFFBemFPOEIyakRsdXEwQT09

Meeting ID: 837 0028 2246
Passcode: ks@2021
One tap mobile
+12532158782,,83700282246#,,,,*7928507# US (Tacoma)
+13017158592,,83700282246#,,,,*7928507# US (Washington DC)

Dial by your location
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 646 558 8656 US (New York)
        +1 669 900 9128 US (San Jose)
Meeting ID: 837 0028 2246
Passcode: 7928507
Find your local number: https://us02web.zoom.us/u/kdAcalqbev

**Comment by shenhonglei** [comment]
Support,It’s an honor to participate and learn with everyone.

**Comment by LinuxSuRen** [comment]
[summer-ospp](https://github.com/summer-ospp/help) is a local event that is similar to [Google Summer of Code](https://github.com/kubesphere/community/issues/280). This could be a potential topic.

**Comment by webup** [comment]
Count me in!

**Comment by LinuxSuRen** [comment]
There's a bunch of proposals that exist. We can the whole list by GitHub label. See also https://github.com/kubesphere/prow-tutorial/pull/24

**Comment by LinuxSuRen** [comment]
The [meeting ntoes](https://docs.google.com/document/d/1amXxLZFkR_4yQU89EZlO_P0fgUVyhXoLmNqvlA55QEg/edit#) is here.

**Comment by FeynmanZhou** [comment]
> [summer-ospp](https://github.com/summer-ospp/help) is a local event that is similar to [Google Summer of Code](https://github.com/kubesphere/community/issues/280). This could be a potential topic.

@LinuxSuRen  I assume register the GSoC first.

**Comment by LinuxSuRen** [comment]
> @LinuxSuRen I assume register the GSoC first.

Of course.

**Comment by rayzhou2017** [comment]
Great! totally support

**Comment by FeynmanZhou** [comment]
@kubesphere/toc Please vote this proposal with your comment or emoji.

**Comment by zheng1** [comment]
agree +1

**Comment by leoendless** [comment]
agree +1

**Comment by benjaminhuo** [comment]
+1

**Comment by LinuxSuRen** [comment]
Thanks for all to support this idea proposal. I'm going to push it forward.
